rpc/cloudagent/cloud/controlplane/controlplane.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
+syntax = "proto3";
+option go_package = "github.com/microsoft/moc/rpc/cloudagent/cloud";
+package moc.cloudagent.controlplane;
+
+import "google/protobuf/wrappers.proto";
+import "common.proto";
+
+message ControlPlaneRequest {
+	repeated ControlPlane ControlPlanes = 1;
+	Operation OperationType = 2;
+}
+
+message ControlPlaneResponse {
+	repeated ControlPlane ControlPlanes = 1;
+	google.protobuf.BoolValue Result = 2;
+	string  Error = 3;
+}
+
+message ControlPlane {
+	string id = 1;
+	string name = 2;
+	string locationName = 3;
+	string fqdn = 4;
+	int32 port = 5;
+	string certificate = 6;
+	int32  authorizerPort = 7;
```

Comment:
What is this one?

Reply:
cert and authPort were planned to provide analagous behavior to node, but are not needed and have been removed.